diaphora.py

```diff
@@ -103,7 +106,7 @@ def ast_ratio(ast1, ast2):
 #-------------------------------------------------------------------------------
 def log(msg):
   if isinstance(threading.current_thread(), threading._MainThread):
-    print(("[%s] %s" % (time.asctime(), msg)))
+    print(("[diaphora][%s] %s" % (time.asctime(), msg)))
```

Comment:
I'm not sure I like adding the 'diaphora' prefix. Are you doing this to be able to discern the output of this or that part inside your tool (idahunt)?

Reply:
Ya, to be honest it is a personal preference here, due to using loads of IDA plugins and scripts. And it makes it a lot easier to know what is logging in the IDA output window (when using the GUI) or in a log file (when using idahunt). I'll make it [diaphora] for both files at least so it is better for you, feel free to ignore otherwise.

Reply:
It sounds OK to me. If it is helpful, it doesn't hurt.